### ISSUE TITLE
Neuropathic pain

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10141]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
The International Association on the Study of Pain (IASP): Pain caused by a lesion or disease of the somatosensory nervous system.

Note: **Neuropathic pain** is a clinical description (and not a diagnosis) which requires a demonstrable lesion or a disease that satisfies established neurological diagnostic criteria. The term lesion is commonly used when diagnostic investigations (e.g. imaging, neurophysiology, biopsies, lab tests) reveal an abnormality or when there was obvious trauma. The term disease is commonly used when the underlying cause of the lesion is known (e.g. stroke, vasculitis, diabetes mellitus, genetic abnormality).
Somatosensory refers to information about the body per se including visceral organs, rather than information about the external world (e.g., vision, hearing, or olfaction). The presence of symptoms or signs (e.g., touch-evoked pain) alone does not justify the use of the term neuropathic. Some disease entities, such as trigeminal neuralgia, are currently defined by their clinical presentation rather than by objective diagnostic testing. Other diagnoses such as postherpetic neuralgia are normally based upon the history. It is common when investigating neuropathic pain that diagnostic testing may yield inconclusive or even inconsistent data. In such instances, clinical judgment is required to reduce the totality of findings in a patient into one putative diagnosis or concise group of diagnoses.

PMID: 28161028 "Traumatic SI (Spinal injury) can give rise to a range of neurological problems, including motor and sensory function loss, intestinal and bladder dysfunction, spasticity, **neuropathic pain** and autonomic dysreflexia."

PMID: 37761417 "Pain management in patients with SCIs can be a challenging matter, and it is best to have an experienced pain-management team in these cases. Although it is always better to start with medications that do not impair respiratory drive and to avoid the ones that can cause cardiac and behavioral complications, many patients will need medications targeting **neuropathic pain**"

PMID: 32859390 "Spinal cord injury pain encompasses musculoskeletal and **neuropathic pain**. Its management is often multidisciplinary and involves specific drugs such as antidepressants and antiepileptics, and nonpharmacological treatment including psychotherapy, physical therapy and neuromodulation techniques. Recent progress in the diagnosis, assessment, and understanding of its mechanisms offers the perspective of a more rational therapeutic management, which should result in better therapeutic outcome."

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Spinal Cord Injury ORPHA:90058

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000040
- label: Neuropathic pain
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: PMID:19400724
- def: Neuropathic pain is triggered by lesions to the somatosensory nervous system that alter its structure and function so that pain occurs spontaneously and responses to noxious and innocuous stimuli are pathologically amplified.
- comment: Neuropathic pain can be characterized by pain that is triggered when clothes touch the skin, spontaneous burning that feels like boiling water, bursts of pins and needles(paresthesias) in the feet upon walking , a continuous crushing pain after an amputation as if your phantom foot is being squeezed, or a band of searing pain around the body at the level at whichsensation was lost after a spinal cord injury.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10141
- synonyms:
